source/viz.md

```diff
@@ -1086,15 +1147,15 @@ The default label is the name of the column being mapped to `color`. Here that
 would be `landmass_type`;
 however `landmass_type` is not proper English (and so is less readable).
 Thus we use the `title` argument inside `alt.Color` to change that to "Type"
-Finally, we again use the `configure_axis` function 
+Finally, we again use the `configure_axis` function
 to change the font size.
 
 ```{code-cell} ipython3
-islands_plot_sorted = alt.Chart(islands_top12).mark_bar(color='black').encode(
-    x = alt.X("size",title = "Size (1000 square mi)"),
-    y = alt.Y("landmass", title = "Landmass", sort='x'),
-    color = alt.Color("landmass_type", title = "Type")).configure_axis(
-    titleFontSize=12)
+islands_plot_sorted = alt.Chart(islands_top12).mark_bar().encode(
+    x=alt.X("size",title="Size (1000 square mi)"),
+    y=alt.Y("landmass", title="Landmass", sort='x'),
+    color=alt.Color("landmass_type", title="Type")
+).configure_axis(titleFontSize=12)
```

Comment:
I am inclined to remove the use of `configure_axis` here. The default font size is reasonable